mappings/net/minecraft/entity/passive/TadpoleEntity.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/unmapped/C_gdxvnoho net/minecraft/entity/passive/TadpoleEntity
+	FIELD f_epyeeylc tadpoleAge I
+	FIELD f_jetulgup GROWTH_AGE I
+	FIELD f_lhkulahc MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_rfntsubn SENSORS Lcom/google/common/collect/ImmutableList;
+	METHOD m_dsreleic decrementStack (Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_sddaxwyk;)V
+		ARG 1 player
+		ARG 2 stack
+	METHOD m_exzbgvfj isFood (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
+		ARG 1 stack
+	METHOD m_fpexwmxb setAge (I)V
+		ARG 1 age
+	METHOD m_iieprzwi growthSeconds (I)V
+		ARG 1 seconds
+	METHOD m_jlrrvacd getTadpoleAge ()I
+	METHOD m_nktwhaku timeTillGrown ()I
+	METHOD m_owndoqec createTadpoleAttributes ()Lnet/minecraft/unmapped/C_sdjuuzrz$C_tehwrjus;
+	METHOD m_pkfuobvm growIntoFrog ()V
+	METHOD m_vclsgbpk interactWithFood (Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_sddaxwyk;)V
```

Comment:
`eatFood`? Do frogs not eat food in a normal way? "interacting" with food doesn't sound right in the slightest

Reply:
They may have other interactions with food in this method in general, how they will eat it stuff like that. eatFood works depending on what is put in this method however. I havent looked at the class so im not totally sure

Reply:
if the only thing this method does is eat the food, then it will need to be changed

Reply:
```suggestion
	METHOD m_vclsgbpk feed (Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_sddaxwyk;)V
```